components/molecules/PaginationResults/pagination-result.tsx

```diff
@@ -19,10 +19,10 @@ const PaginationResults = ({ metaInfo, entity, total, className }: PaginationRes
       ? metaInfo.page * metaInfo.limit
       : metaInfo.itemCount;
   return (
-    <div className={`${className && className} flex items-center gap-x-1   text-sm text-light-slate-9 tracking-tight`}>
-      <span className="hidden md:block">Showing</span> <span className="text-light-slate-12">{from} -</span>
+    <div className={`${className && className} md:inline-block items-center gap-x-1   text-sm text-light-slate-9 tracking-tight`}>
+      <span className="hidden md:inline-block">Showing</span> <span className="text-light-slate-12">{from} -</span>
       <span className="text-light-slate-12">{to}</span> of {total > 999 ? humanizeNumber(total, null) : total}
-      <span>{entity}</span>
+      <span> {entity}</span>
```

Comment:
We should hide this entity on medium screens only

Reply:
Thank you for the feedback, I did that in the last Commit and added a video.